Replace trait chart tooltip with foldable list

christoph-frick opened the pull request:
This makes it easier to check the cards from a trait
- can be clicked for the full view
- more room to display icon, cost, ...

fspoettel commented:
Very nice idea!

I gave this a shot and have a few usability notes:
- I think the concise hover tooltip should stay when the row is collapsed. When it's my own deck, I don't need to see the cards sometimes, the names are then enough.
- rather than clicking the `v` indicator, the whole row should be clickable to expand.
- when a row is expanded, it should not have a hover background color change should not apply to all of the children.
- it would be nice if the cards had tooltips - this might be as simple as replacing the `ListCardInner` component with `ListCard`.
- There is a text alignment issue between the `Trait` and `Count` (I think caused by the expand button?)
- The component should take a bit more vertical space now. If it no longer works in the chart grid, I would be fine with making it more prominent and moving it to its own sections like the Limited slot display.

christoph-frick commented:
Thank you for the review.

I have addressed the following points:

- smaller button for the detail-fold moves trait and count back on the
  same baseline (and makes more lines-per-hight visible)
- increased height of the table (250px → 330px, roughly a line more than
  the pie chart)
- right align count column content
- tooltip is back (on the trait-`span` only)
- trait-`span` is also part of the fold-toggle (unless you really meant
  the whole row for toggling)
- detail cards rendered using `ListCard` to have a tooltip
- when unfolded, disable hover-indicator
- fixed linter errors (formatting, and `cardLevelDisplay` for the
  details)

fspoettel commented:
Thanks for the PR @christoph-frick, great work! 🙌
